apollo-integration/src/test/kotlin/com/apollographql/apollo/ApolloWatcherTest.kt

```diff
@@ -19,7 +19,7 @@ import com.apollographql.apollo.integration.normalizer.EpisodeHeroNameWithIdQuer
 import com.apollographql.apollo.integration.normalizer.HeroAndFriendsNamesWithIDsQuery
 import com.apollographql.apollo.integration.normalizer.StarshipByIdQuery
 import com.apollographql.apollo.integration.normalizer.type.Episode
-import com.apollographql.apollo.internal.RealApolloStore
+import com.apollographql.apollo.cache.normalized.internal.RealApolloStore
```

Comment:
👍  Makes sense to have the store outside the runtime. If it grows, we could make it a separate module but I don't think it's worth it given it's pretty small-ish right now